de.ient.YUView.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Exec=YUView %F
+Name=YUView
+GenericName=YUV Video Player
+X-KDE-StartupNotify=true
+Icon=de.ient.YUView
+Terminal=false
+Categories=AudioVideo
+MimeType=application/vnd.ient.yuview.yuvfile
```

Comment:
You will need a extra mime file to make this work, e.g.: https://github.com/flathub/com.jetbrains.PyCharm-Community/blob/master/pycharm.xml

Reply:
Can I remove the MimeType field and just create the xml as in pycharm? pycharm also doesn't have a MimeType in the desktop entry

Reply:
how does it derive the correct app. in the pycharm xml
```
application/PyCharm
```
but in the corresponding desktop file
```
Name=PyCharm Community Edition
GenericName=PyCharm Community Edition - The intelligent Python IDE
Exec=pycharm-desktop
```
souldn't those match?